pkgs/development/python-modules/setuptools/default.nix

```diff
@@ -27,14 +27,22 @@ let
       name = "${pname}-${version}-source";
     };
 
+    patches = [
+      ./tag-date.patch
+    ];
+
     buildPhase = ''
       ${python.pythonForBuild.interpreter} bootstrap.py
       ${python.pythonForBuild.interpreter} setup.py sdist --formats=gztar
     '';
 
+    # Here we untar the sdist and retar it in order to control the timestamps
+    # of all the files included
     installPhase = ''
-      echo "Moving sdist..."
-      mv dist/*.tar.gz $out
+      echo "Repackaging sdist..."
```

Comment:
please do the repackaging in the `buildPhase`, and keep the `mv` in `installPhase`. It's good practice to keep the two steps in distinct phases.

Reply:
Sure I can do that :+1: